packages/fluent-editor/src/table/modules/table-operation-menu.ts

```
@@ -362,7 +362,15 @@ export default class TableOperationMenu {
       'width': `${MENU_WIDTH}px`,
     }
 
-    if (menuHeight + top > winHeight && menuHeight < winHeight) {
+    if (menuHeight + top > winHeight) {
+      delete cssContent.top
+      cssContent.bottom = '10px'
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)